app/src/main/java/com/evercheck/flagly/utils/CoroutineContextProvider.kt

```diff
@@ -1,8 +1,9 @@
 package com.evercheck.flagly.utils
 
+import javax.inject.Inject
 import kotlinx.coroutines.CoroutineDispatcher
 
-class CoroutineContextProvider(
+class CoroutineContextProvider @Inject constructor (
     val mainDispatcher: CoroutineDispatcher,
     val backgroundDispatcher: CoroutineDispatcher
```

Comment:
I would rename this, `io`, is ok and smaller, in wallet we use `io` too :)

Reply:
Done